phpBB/viewonline.php

```diff
@@ -357,6 +355,13 @@
 		default:
 			$location = $user->lang['INDEX'];
 			$location_url = append_sid("{$phpbb_root_path}index.$phpEx");
+
+			if ($row['session_page'] === 'app.' . $phpEx . '/help/faq' ||
```

Comment:
To be honest, we should probably add a dedicated class/method that takes care of this for valid routes. This is out of the scope of this PR though ;)